File: src/main/java/id/hackathonmerdeka/hackmdk3/controller/LoginController.java
package id.hackathonmerdeka.hackmdk3.controller;

import id.hackathonmerdeka.hackmdk3.model.User;
import id.hackathonmerdeka.hackmdk3.repository.UserRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.RestController;

/**
 * @author Arthur Purnama (arthur@purnama.de)
 */
@RestController
public class LoginController extends PublicController {

	@Autowired
	private UserRepository repository;

	@RequestMapping(path="/login", method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
	public Iterable<User> findAll(){
		return repository.findAll();
	}

}

File: src/main/resources/static/js/controller/LoginController.js
/**
 * @author Arthur Purnama (arthur@purnama.de)
 */
birokrazyApp.controller('LoginController', ['$scope', '$rootScope', '$location',
    function ($scope, $rootScope, $location) {
    if($rootScope.authenticated){
        $location.path("/user/application");
    }
}]);